apps/src/templates/sectionProgress/progressTables/ProgressTableView.jsx

```diff
@@ -356,7 +356,7 @@ class ProgressTableView extends React.Component {
         </div>
         {this.props.currentView === ViewType.DETAIL ? (
           <ProgressLegend
-            excludeCsfColumn={!this.props.scriptData.csf}
+            includeCsfColumn={this.props.scriptData.csf}
```

Comment:
This looks like it's missing the `includeReviewState` prop

Reply:
i wasn't sure what the status of that prop is at this point so i figured wiring up these changes could be done in whatever PR brings that prop into this component (probably LP-1901?). i'm happy to do some of that in this PR if you'd like, but we should sync first